console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -15,22 +16,18 @@ import {
   Button,
   ButtonVariant
 } from "@patternfly/react-core";
-import { Loading } from "use-patternfly";
-import { useQuery } from "@apollo/react-hooks";
 import { OutlinedCopyIcon } from "@patternfly/react-icons";
 import { StyleSheet, css } from "@patternfly/react-styles";
-import AceEditor from "react-ace";
-import "ace-builds/src-noconflict/mode-java";
-import "ace-builds/src-noconflict/theme-github";
-import { ADDRESS_SPACE_COMMAND_REVIEW_DETAIL } from "graphql-module/queries";
 
-export interface IAddressSpaceReview {
+export interface IAddressSpaceReviewProps {
   name?: string;
   type?: string;
   plan?: string;
   namespace: string;
   authenticationService: string;
+  data: any;
```

Comment:
I think we can replace any with some interface. As we have to avoid any as type.

Reply:
changes done